typescript/sdk/src/token/TokenStandard.ts

```diff
@@ -53,6 +55,8 @@ export const TOKEN_STANDARD_TO_PROTOCOL: Record<TokenStandard, ProtocolType> = {
   EvmHypCollateral: ProtocolType.Ethereum,
   EvmHypcollateralVault: ProtocolType.Ethereum,
   EvmHypSynthetic: ProtocolType.Ethereum,
+  EvmHypXERC20Collateral: ProtocolType.Ethereum,
+  EvmHypFiatCollateral: ProtocolType.Ethereum,
```

Comment:
I think you'll need to hook these new standards up to a token adapter in `Token.ts#getHypAdapter`